cdap-common/src/main/java/io/cdap/cdap/common/messaging/MessagingUtils.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package io.cdap.cdap.messaging;
+package io.cdap.cdap.common.messaging;
```

Comment:
It looks like this class is used in cdap-tms and cdap-watchdog. cdap-watchdog has tms as dependency. Can we move it to cdap-tms and leave in the original package?